src/NzbDrone.Common/Instrumentation/Sentry/SentryTarget.cs

```diff
@@ -115,6 +115,11 @@ public SentryTarget(string dsn)
                                      });
             
             _debounce = new SentryDebounce();
+
+            // initialize to true and reconfigure later
+            // Otherwise it will default to false and any errors occuring
+            // before config file gets read will not be filtered
+            FilterEvents = true;
```

Comment:
Could also be an issue the other way around if you don't want things filtered, but then they do on startup. I think setting to `true` to filter on startup is bigger priority, so I'm good with this change.